lib/telegraf/telegraf.conf.erb

```diff
@@ -798,20 +798,20 @@
   ##   withdbname boolean
   ##   tagvalue string (comma separated)
   ##   measurement string
-  <% server_part = server_url.split("@").last.split("?").first.strip %>
-  <% server_part = ".#{server_part}" unless server_part.empty? %>
   [[inputs.postgresql_extensible.query]]
     sqlquery="SELECT * FROM pg_stat_database WHERE datname"
     version=901
     withdbname=true
     tagvalue=""
-    measurement="postgresql<%= server_part %>"
+    measurement=""
   [[inputs.postgresql_extensible.query]]
     sqlquery="SELECT * FROM pg_stat_bgwriter"
     version=901
     withdbname=false
     tagvalue=""
-    measurement="postgresql<%= server_part %>"
+    measurement=""
+  [inputs.postgresql_extensible.tags]
+    server_name = <%= server_url.split("@").last.split("?").first.strip.inspect %>
```

Comment:
The format is explained in the comment above: https://github.com/Instrumental/instrumentald/pull/16/files/04fddb499683de242e812d39ec69d8c432811dbd#diff-1ea3802849cf2629bd2f3ef86bfa22c2R755

I can re-explain it here in ruby comment. Want?